zerver/views/messages.py

```diff
@@ -295,7 +295,7 @@ def by_pm_with(self, query: Query, operand: str, maybe_negate: ConditionTransfor
                 emails = [e.strip() for e in operand.split(',') if e.strip() != self.user_profile.email]
                 recipient = recipient_for_emails(emails, False,
                                                  self.user_profile, self.user_profile)
-            except ValidationError:
+            except JsonableError:
```

Comment:
This is a little unfortunate, since `ValidationError` still would have made sense in this particular context.  Part of me is wondering if we should just fully decouple `recipient_for_emails` from the `Addressee` code.